core/templates/dev/head/pages/exploration-editor-page/improvements-tab/templates/answer-details-modal.template.html

```diff
@@ -44,7 +44,7 @@ <h3 class="modal-title">
     </div>
   </div>
   <div class="modal-footer">
-    <button ng-disabled="!selectedLearnerAnswerInfo.length"
+    <button ng-disabled="!isUserLoggedIn || !selectedLearnerAnswerInfo.length"
```

Comment:
The button should not be shown at all (rather than disabled) if the user is not logged in.

Reply:
This (and other comments) can be done but it will likely result in a poor UX. Let's save that for a separate issue to prioritize fixing the bug?